frontend/package.json

```diff
@@ -102,6 +106,8 @@
     "semver": "^7.3.8",
     "string-hash": "^1.1.3",
     "superstruct": "^1.0.3",
+    "tailwind-merge": "^2.2.1",
```

Comment:
If possible, we can avoid this and create our own util function

Reply:
Will be keeping this